libraries/libfc/src/crypto/elliptic_secp256k1.cpp

```diff
@@ -113,20 +114,29 @@ namespace fc { namespace ecc {
       return my->_key != empty_pub;
     }
 
+    std::string public_key::to_base58() const
+    {
+        FC_ASSERT( my->_key != empty_pub );
+        return to_base58( my->_key );
+    }
+
     public_key_data public_key::serialize()const
     {
         FC_ASSERT( my->_key != empty_pub );
         return my->_key;
     }
-
+    // TODO:  @jglanz OpenSSL is implemented via boringssl in all versions of EOSIO, and does not SEEM to provide
+    //   `EC_KEY` curve related implementations, but I need to investigate.
+    //   this should likely be swapped out the same way that I resolved `fc::em::public_key`
+    //   support, via `libsecp256k1`
```

Comment:
boringssl was a fairly recent change. It used openssl before. The boringssl was used if I remember correctly to provide portable execs.

This TODO needs addressed.